### PR TITLE
Add fuzzing automation test script

### PR DESCRIPTION
**Pull request by Zhiqiang520**
Add fuzzing automation test script
Signed-off-by: Zhao, Zhiqiang <zhiqiang.zhao@intel.com>

### CODE REVIEW

File: unit_test/fuzzing/fuzzing.sh [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+if [[ $2 = "ON" ]]; then
+    cd $fuzzing_out
+    mkdir coverage_log
+    cd coverage_log
+    lcov --capture --directory $libspdm_path --output-file coverage.info
+    genhtml coverage.info --output-directory .
+fi
+
+cd $script_path
+python3 ./summary.py $fuzzing_out
```
Comment (jyao1) [inline unit_test/fuzzing/fuzzing.sh:157]: Do we must use python? Can we just use normal shell script?
If have to, then we need document python requirement, e.g python3 version.

Comment (Zhiqiang520) [reply unit_test/fuzzing/fuzzing.sh]: Python is not necessary, and we can just use the normal shell script. I have updated.

Comment (jyao1) [reply unit_test/fuzzing/fuzzing.sh]: Thank you!

**Comment by jyao1** [comment]
looks good

**Pull request merged**